h/static/scripts/widget-controller.js

```diff
@@ -167,6 +197,14 @@ module.exports = function WidgetController(
     }
   }
 
+  function selectAppropriateTab() {
+    if (annotationUI.getState().selectedTab === uiConstants.TAB_ORPHANS) {
+      annotationUI.selectTab(uiConstants.TAB_ANNOTATIONS);
+    } else {
+      annotationUI.selectTab(annotationUI.getState().selectedTab);
```

Comment:
This line of code doesn't do anything, or at least, I wouldn't expect it to, since all that the `SELECT_TAB` action does is updated the `selectedTab` state variable. If it _is_ doing something, that's a bug and we need to understand what is going on.